File: 2.0 Conditional Statements - More Exercises/08. Fuel Tank.js
function fuelTank(typeFuel, quantity) {
    const lst = ['Diesel', 'Gasoline', 'Gas']

    if (!lst.includes(typeFuel))
        return 'Invalid fuel!'
    if (quantity >= 25)
        return `You have enough ${typeFuel.toLowerCase()}.`
    if (quantity < 25)
        return `Fill your tank with ${typeFuel.toLowerCase()}!`
}

// console.log(fuelTank('Diesel', 10))   //Fill your tank with diesel!

File: PB - More Exercises Python/2.0 Conditional Statements - More Exercises/08. Fuel Tank.py
type_fuel, quantity = input().lower(), int(input())

if type_fuel not in ['diesel','gasoline','gas']: print('Invalid fuel!')
elif quantity >= 25: print(f'You have enough {type_fuel}.')
else: print(f'Fill your tank with {type_fuel}!')

#-------------------------------------------(2)------------------------------------------
    

type_fuel, quantity = input(), int(input())
lst = ['Diesel','Gasoline','Gas']

if type_fuel not in lst:
    print('Invalid fuel!')
elif quantity >= 25:
    print(f'You have enough {type_fuel.lower()}.')
elif quantity < 25:
   print(f'Fill your tank with {type_fuel.lower()}!')


'''
Diesel
10
'''
